### PR TITLE
adding myself to committers.md

### PR DESCRIPTION
**Pull request by shaneknapp**
EOM

### CODE REVIEW

**Review by gatorsmile — COMMENTED** [review]
LGTM. Merged to master.

File: committers.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ navigation:
 |Shane Huang|Intel|
 |Dongjoon Hyun|Hortonworks|
 |Holden Karau|Google|
+|shane knapp|UC Berkeley|
```
Comment (gatorsmile) [inline committers.md:38]: Congratulations!

**Comment by shaneknapp** [comment]
stupid question:  should i have run the jekyll build and included the changes to `committers.html`?

**Comment by shaneknapp** [comment]
@gatorsmile

**Comment by holdenk** [comment]
@shaneknapp yeah, we don't automatically build the markdown for serving it's checked into the repo as static HTML.

**Comment by shaneknapp** [comment]
ayup, figured as much.  i'll push an update out a little later today.

**Comment by gatorsmile** [comment]
It has been updated. Thanks!

**Comment by shaneknapp** [comment]
yep, i just saw that someone else actually followed the instructions and i
rode on their coattails.  :)

**Pull request merged**